docs/nodes/build/run-avalanche-node-manually.md

```diff
@@ -108,7 +108,7 @@ Download the AvalancheGo repository into your `$GOPATH`:
 cd $GOPATH
 mkdir -p src/github.com/ava-labs
 cd src/github.com/ava-labs
-git clone git@github.com:ava-labs/avalanchego.git
+git clone https://github.com/ava-labs/avalanchego
```

Comment:
Most uses ssh, not https: https://www.howtogeek.com/devops/should-you-use-https-or-ssh-for-git/